### DISCUSSION TITLE
Ready to use components

### DISCUSSION THREAD

**Discussion by britisharmy** [lit/lit#3979] [Ideas]
I have been creating some ui components and i was thinking of sharing them.  Some components have props some don't. I think they shall save time for some tasks. I have styled the components using bootstrap 5.3 but you can write your own styles.

Is there a list of community components i can add to?

**Comment by tbroyer** [comment]
There's https://www.webcomponents.org/ (which is being rebuilt at https://github.com/webcomponents/webcomponents.org)

**Reply by britisharmy** [reply]
Oh nice. I am having a look at them.

**Comment by britisharmy** [comment]
This are my components https://github.com/britisharmy/lit_components
